source/middleman.html.md.erb

```diff
@@ -0,0 +1,16 @@
+---
+title: Middleman options
+weight: 87
+---
+
+# Middleman options
+
+The GDS technical documentation tool is built with
+[Middleman](https://middlemanapp.com/) [external link], so it offers all the same
+functionality. Much of this is not specific to the tool itself.
+
+For example, you can read Middleman's own documentation about [external links]:
```

Comment:
Is that the accepted style for multiple external links in a list? Usually, I would put [external link] after each bullet point. Otherwise it kind of looks like [external links] is what is being referred to. Which it kind of is but you get my point.

Reply:
looks good :-)